dev/scripts/sortPackageJson.js

```diff
@@ -0,0 +1,73 @@
+#! /usr/bin/env node
+
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+/**
+ * This script sorts:
+ * 
+ * package.json
+ *  - activationEvents
+ *  - contributes.commands
+ *  - contributes.menus
+ * 
+ * package.nls.json
+ */
+
+const path = require('path');
+const existsSync = require('fs').existsSync;
+const fs = require('fs/promises');
+
+async function sortPackageJson(path) {
+    if (!existsSync(path)) {
```

Comment:
If you change the import statement:

```suggestion
    if (!fs.existsSync(path)) {
```